### PR TITLE
files preview: truncate file names

### PR DESCRIPTION
**Pull request by jrcastro2**
* closes https://github.com/inveniosoftware/invenio-app-rdm/issues/645

image

### CODE REVIEW

**Review by fenekku — APPROVED** [review]
Some merge conflicts, but once resolved it will be good to merge.

**Comment by jrcastro2** [comment]
> Some merge conflicts, but once resolved it will be good to merge.

Done.

File: invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/landing_page.less [hunk 1]
```diff
@@ -164,6 +164,7 @@
      .content{
          padding-top: 0px !important;
      }
+     -K|--keep-services)
```
Comment (fenekku) [inline invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/landing_page.less:167]: Oups?

Comment (jrcastro2) [reply invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/landing_page.less]: Wow, nice catch, thanks! My brain is already on holidays mode ... :sweat_smile:

**Review by fenekku — APPROVED** [review]

**Pull request merged**